packages/fluentui/docs/src/prototypes/table/ResponsiveTable.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { TableCell, TableProps, Table } from '@fluentui/react-northstar';
+import calculateBreakpoints, { Column } from '../../utils/calculateBreakpoints';
+import * as _ from 'lodash';
+
+interface ResponsiveTableProps extends TableProps {
+  config: Column[];
+  children: React.ReactElement<typeof Table>;
+  breakpoints?: number[];
+  id?: string;
+}
+
+const ResponsiveTableContainer: React.FC<ResponsiveTableProps> = props => {
+  const { config, children: ChildrenTable } = props;
```

Comment:
```suggestion
  const { config, children } = props;
```
```suggestion
  const { config, children: childrenTable } = props;
```

Not sure about naming there. Pascalcase is used for element types, not for elements.

Reply:
Done